hyperspy/_signals/signal1d.py

```diff
@@ -98,9 +98,10 @@ def find_peaks_ohaver(y, x=None, slope_thresh=0., amp_thresh=None,
         contains position, height, and width of each peak
     Examples
     --------
-    >>> x = arange(0,50,0.01)
-    >>> y = cos(x)
-    >>> one_dim_findpeaks(y, x,0,0)
+    >>> import numpy as np
```

Comment:
There is no need to import numpy, the fixture does it and the User Guide explains that all the examples assume that plt, hs and np are in the namespace.